mm2src/mm2_main/tests/docker_tests_sia_unique.rs

```diff
@@ -1,12 +1,9 @@
 #![allow(unused_imports, dead_code)]
-#![cfg(feature = "enable-sia")]
```

Comment:
Why the removal of `enable-sia` from this file? It's only related to sia feature

Reply:
Probably by mistake during fixing all the errors and warnings, will revert it.